WordPress/src/main/java/org/wordpress/android/ui/posts/PostSettingsTagsActivity.java

```diff
@@ -30,6 +30,7 @@
 import org.wordpress.android.fluxc.model.TermModel;
 import org.wordpress.android.fluxc.store.TaxonomyStore;
 import org.wordpress.android.util.ToastUtils;
+import org.wordpress.android.util.WPActivityUtils;
```

Comment:
It was, one of them doesn't need the view but not as efficient. We could use that one in both places too, if it's a problem.